docs/hosting-a-server-with-northstar/troubleshooting.md

```diff
@@ -45,11 +28,11 @@ This won't generally help but will allow you to avoid checking for the wrong ser
 
 Check if your server is reachable from your internal network's IP (often starts with `192.168.x.x`)
 
-**Firewall is blocking tcp ports**
+**Firewall is blocking ports**
 
 In some cases your Firewall or antivirus can prevent your ports to be exposed to your local network.
 To fix this issue, make a rule to allow your server to listen on your network.
-Disabling the firewall and antivirus can also work, even if it's not reccomended.
+Disabling the firewall and antivirus can also work, even if it's not recommended.
```

Comment:
Unrelated to PR, reeeeeeeeee

(I'll allow it but this really annoys me >.<)

Reply:
I couldn't resist that one.